src/core_plugins/tile_map/public/kibana_map.js

```diff
@@ -309,15 +308,8 @@ export class KibanaMap extends EventEmitter {
 
     const southEast = bounds.getSouthEast();
     const northWest = bounds.getNorthWest();
-    let southEastLng = southEast.lng;
-    if (southEastLng > 180) {
-      southEastLng -= 360;
```

Comment:
Personally not a fan of default params, since they hurt searchability in the code, especially in a language like JS. Would add a parallel method, getBoundsTrimmed, or getBoundsWithoutTrimming, whatever would require the least changes.

Anyways, it's your call ;)  as long as we avoid regressing on https://github.com/elastic/kibana/issues/8946.